vagrant/provisioning/roles/elasticsearch-install/tasks/main.yml

```diff
@@ -33,15 +33,21 @@
         dest: /etc/elasticsearch
         backup: yes
     
-    - name: Adding services to auto start
+    - name: ElasticSearch systemd unit file
+      become: yes
+      template:
+        src: "elasticsearch.service"
+        dest: /etc/systemd/system/elasticsearch.service
+      register: elasticsearch_systemd
+
+    - name: enable ElasticSearch to start on boot
+      become: yes
       systemd:
         name: elasticsearch
         enabled: yes
-        daemon_reload: yes
         masked: no
-        state: started
```

Comment:
@bojmil-armedia , if you exclude state: started, than elasticsearch will not start . Only it will be enabled for the next boot.
Also i suggest you add another task before this one for reload daemon if needed since with copying our elastisearch.service template into /etc/systemd/system/elasticsearch.service we did changes and we must reload daemon.
For ex. :
- name: Reload daemon files if needed
  become: yes
  command: systemctl daemon-reload
  when: elasticsearch_systemd is changed

And after that if state is = started then elasticsearch will start running.

Reply:
Right. Not sure when I deleted this line.